content/build/upload/turbo-credits.mdx

```diff
@@ -27,37 +27,33 @@ Turbo Credits represent upload power on the Arweave network, divisible down to 1
 
 ### Payment Methods
 
-- **Fiat Currency**: Credit/debit cards via Stripe
-- **Crypto Tokens**: AR, ETH, SOL, MATIC, ARIO, USDC, KYVE, ETH (BASE)
-- **Multiple Wallets**: Ethereum, Solana, and Arweave wallets supported
+- **Fiat Currency**: Credit/debit cards or USDC via Stripe
+- **Crypto Tokens**: AR, ETH, SOL, POLYGON, ARIO, USDC, KYVE via transfer transactions
+- **Multiple Wallets**: Ethereum, Solana, Arweave, and Kyve wallets supported
 
 ### Supported Tokens & Purchase Methods
 
-| Payment Method               | Turbo SDK | Turbo CLI | Turbo API | Top Up App | ArDrive App |
+| Payment Method               | Turbo SDK | Turbo CLI | Turbo API | Turbo App  | ArDrive App |
 | ---------------------------- | --------- | --------- | --------- | ---------- | ----------- |
 | **Fiat (credit/debit card)** | ✅        | ✅        | ✅        | ✅         | ✅          |
 | **AR**                       | ✅        | ✅        | ✅        | ✅         | ❌          |
 | **ETH**                      | ✅        | ✅        | ✅        | ✅         | ❌          |
+| **ETH (BASE)**               | ✅        | ✅        | ✅        | ✅         | ❌          |
 | **SOL**                      | ✅        | ✅        | ✅        | ✅         | ❌          |
-| **MATIC**                    | ✅        | ✅        | ✅        | ❌         | ❌          |
+| **POLYGON**                  | ✅        | ✅        | ✅        | ✅         | ❌          |
+| **ARIO**                     | ✅        | ✅        | ✅        | ✅         | ❌          |
+| **USDC (ETH)**               | ✅        | ✅        | ✅        | ❌         | ❌          |
+| **USDC (BASE)**              | ✅        | ✅        | ✅        | ❌         | ❌          |
+| **USDC (POLYGON)**           | ✅        | ✅        | ✅        | ❌         | ❌          |
 | **KYVE**                     | ✅        | ✅        | ✅        | ❌         | ❌          |
-| **ETH (BASE)**               | ✅        | ✅        | ✅        | ❌         | ❌          |
-| **ARIO**                     | ✅        | ✅        | ✅        | ❌         | ❌          |
-| **USDC**                     | ❌        | ✅        | ✅        | ❌         | ❌          |
-
-<Callout type="info">
```

Comment:
Perhaps rather than simply excising this exception, we call out the capability explicitly instead? Perhaps up in the Payment Methods section?